core/src/miner/miner.rs

```diff
@@ -524,6 +531,22 @@ impl Miner {
                 // already have transaction - ignore
                 Err(Error::History(HistoryError::TransactionAlreadyImported)) => {}
                 Err(e) => {
+                    match e {
+                        Error::Runtime(RuntimeError::AssetNotFound {
+                            ..
+                        })
+                        | Error::Runtime(RuntimeError::AssetSchemeNotFound {
+                            ..
+                        })
+                        | _ if self
+                            .mem_pool
+                            .read()
+                            .is_local_transaction(hash)
+                            .expect("The tx is clearly fetched from the mem pool") => {}
+                        _ => {
```

Comment:
Other errors are safe to ban the error creators?
And what can we do to reduce some bugs when we add a new RuntimeError?

Reply:
Currently I cannot sure that the left runtime errors are considered as malicious behaviors. It's better to use the blacklist method when some errors are problematic.

Reply:
I picked the two problematic runtime errors `RuntimeError::AssetSupplyOverflow` and `RuntimeError::InvalidScript` and blacklisted them.

Reply:
@foriequal0 @sgkim126 What do you think about judging a malicious user only if the user's transaction failed with the blacklisted error?

Reply:
I think it can be fixed later.